requirements-test-libraries.txt

```diff
@@ -19,11 +22,15 @@ pyzmq==19.0.0
 Unidecode==1.1.1
 zeep==3.4.0
 sentry-sdk==0.19.3
-av==8.0.2
+av==8.0.3
 passlib==1.7.2
 publicsuffix2==2.20191221
 
 # ------------------- Python Version/Platform (OS) specifics
+# No prebuilt wheels for Python 3.9
+h5py==2.10.0; python_version < '3.9'
+h5py==3.2.1; python_version > '3.9'

```

Comment:
One of these two should probably include '='.

Reply:
Yeah, caught that locally